$NetBSD$

Do not hard-code install location
See Github sierrafoxtrot/srecord #68

--- CMakeLists.txt.orig	2022-10-18 10:47:58.000000000 +0000
+++ CMakeLists.txt
@@ -23,16 +23,13 @@ set(CMAKE_BUILD_TYPE Release)
 # Headers are all relative to project root
 include_directories(./)
 
-# Support for pulling in dependencies required for Windows install
-include(InstallRequiredSystemLibraries)
-
 # Support standard install locations
 include(GNUInstallDirs)
 
 # FHS compliant paths for Linux installation
-if(NOT WIN32 AND CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
-#  set(CMAKE_INSTALL_PREFIX "/opt/${PROJECT_NAME}")
-  set(CMAKE_INSTALL_PREFIX "/usr")
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT
+        AND NOT WIN32 AND CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
+  set(CMAKE_INSTALL_PREFIX "/usr" CACHE PATH "Install prefix" FORCE)
 endif()
 
 # Pull in the rest of the pieces
